lale/lib/aif360/urbis.py

```diff
@@ -57,6 +70,8 @@ def __init__(
         estimator,
         unfavorable_labels=None,
         redact=True,
+        imbalance_repair_level=0.8,
```

Comment:
In addition to adding the hyperparameters to the constructor, we'll also need them in the `_hyperparams_schema`.

Reply:
Addressed!